<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QWidget" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>343</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Configuration Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabConfiguration">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_info">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_autoUpdate">
         <item>
          <widget class="QLabel" name="label_autoUpdate">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Auto update interval, msec</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_autoUpdate">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_autoUpdate">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>333</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>500</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_gui">
         <item>
          <widget class="QLabel" name="label_gui">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Path to GUI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_gui"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_gui">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_netctl">
         <item>
          <widget class="QLabel" name="label_netctl">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Path to netctl</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_netctl"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_netctl">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_sudo">
         <item>
          <widget class="QCheckBox" name="checkBox_sudo">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Use sudo for netctl</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_sudo"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_sudo">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_wifi">
         <item>
          <widget class="QCheckBox" name="checkBox_wifi">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Show 'Start WiFi menu'</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_wifi"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_wifi">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showBigInterface">
         <property name="text">
          <string>Show more detailed interface</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showNetDev">
         <property name="text">
          <string>Show network devices</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showExtIp">
         <property name="text">
          <string>Show external IP</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_showIntIp">
         <property name="text">
          <string>Show internal IP</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_configuration">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAppearance">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="layout_font">
         <item>
          <widget class="QLabel" name="label_font">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Font</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_font">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFontComboBox" name="fontComboBox_font">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set font family</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_fontSize">
         <item>
          <widget class="QLabel" name="label_fontSize">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Font size</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_fontSize">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_fontSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set font size</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>7</number>
           </property>
           <property name="maximum">
            <number>48</number>
           </property>
           <property name="value">
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_fontColor">
         <item>
          <widget class="QLabel" name="label_fontColor">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Font color</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_fontColor">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KColorCombo" name="kcolorcombo_fontColor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set font color</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_fontWeight">
         <item>
          <widget class="QLabel" name="label_fontWeight">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Font weight</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_fontWeight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_fontWeight">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set font weight</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>300</number>
           </property>
           <property name="maximum">
            <number>600</number>
           </property>
           <property name="singleStep">
            <number>50</number>
           </property>
           <property name="value">
            <number>400</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_fontStyle">
         <item>
          <widget class="QLabel" name="label_fontStyle">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Font style</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_fontStyle">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_fontStyle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Set font style</string>
           </property>
           <item>
            <property name="text">
             <string>normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>italic</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_appearance">
         <property name="lineWidth">
          <number>5</number>
         </property>
         <property name="midLineWidth">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_activeIcon">
         <item>
          <widget class="QLabel" name="label_activeIcon">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Active icon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_activeIcon"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_activeIcon">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_inactiveIcon">
         <item>
          <widget class="QLabel" name="label_inactiveIcon">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Inactive icon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_inactiveIcon"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_inactiveIcon">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_appearance">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>spinBox_autoUpdate</tabstop>
  <tabstop>lineEdit_gui</tabstop>
  <tabstop>pushButton_gui</tabstop>
  <tabstop>lineEdit_netctl</tabstop>
  <tabstop>pushButton_netctl</tabstop>
  <tabstop>checkBox_sudo</tabstop>
  <tabstop>lineEdit_sudo</tabstop>
  <tabstop>pushButton_sudo</tabstop>
  <tabstop>checkBox_wifi</tabstop>
  <tabstop>lineEdit_wifi</tabstop>
  <tabstop>pushButton_wifi</tabstop>
  <tabstop>checkBox_showBigInterface</tabstop>
  <tabstop>checkBox_showNetDev</tabstop>
  <tabstop>checkBox_showExtIp</tabstop>
  <tabstop>checkBox_showIntIp</tabstop>
  <tabstop>fontComboBox_font</tabstop>
  <tabstop>spinBox_fontSize</tabstop>
  <tabstop>kcolorcombo_fontColor</tabstop>
  <tabstop>spinBox_fontWeight</tabstop>
  <tabstop>comboBox_fontStyle</tabstop>
  <tabstop>lineEdit_activeIcon</tabstop>
  <tabstop>pushButton_activeIcon</tabstop>
  <tabstop>lineEdit_inactiveIcon</tabstop>
  <tabstop>pushButton_inactiveIcon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
